index.html

```diff
@@ -48,5 +48,162 @@ <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
         </div>
       </div>
     </section>
+
+    <section class="project-section">
+      <div class="project-items">
+        <img src="images/portfolio1.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Tonic</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="project-items">
+        <img src="images/portfolio2.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Multi-Post Stories</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
```

Comment:
- [ ] your buttons should follow the button styling: Active, Hover, and Pressed as mentioned in Figma , **_kindly fix that for all of your buttons_**

![image](https://user-images.githubusercontent.com/89282221/192070634-2fd26179-04f5-41b6-8fcf-de0c72b14804.png)